wordsearch.py

```diff
@@ -0,0 +1,44 @@
+# Time Complexity : Exponential but find it difficult to compute
+# Space Complexity : Exponential but find it difficult to compute
+# Did this code successfully run on Leetcode : Yes
+# Any problem you faced while coding this : No
```

Comment:
Look for worst case scenario, you'll be able to figure it out 
for this it's MxN